adapt/intent.py

```diff
@@ -129,23 +135,24 @@ def validate(self, tags, confidence):
         """Using this method removes tags from the result of validate_with_tags
 
         Returns:
-            intent(intent): Resuts from validate_with_tags
+            intent(intent): Results from validate_with_tags
         """
         intent, tags = self.validate_with_tags(tags, confidence)
         return intent
 
-    def validate_with_tags(self, tags, parse_weight):
+    def validate_with_tags(self, tags, confidence):
```

Comment:
It's unfortunate, but this actually constitutes a breaking change (specifically named arguments; caught it when merging some new tests). I have kept all the previous fixes where we were overriding confidence or using the wrong one, and we have scoring tests to validate that.

Reply:
I don't think so; the parser is _so_ lightweight that I'm actually concerned about the perf of something like that. I suppose we could just emit on first use? But I don't think the changes from the PR that introduced `parse_weight` have made it too broadly into the world yet, so its uses should be minimal.